regex-assembly/932290.ra

```diff
@@ -0,0 +1,6 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##! Detection for Bash Quote Evasion
+
+[\"']\w+[\"']\w+
```

Comment:
```suggestion
[\"']\w+[\"']\w+
\w+[\"']\w+[\"']
```

Reply:
Current regex is not catchig this:
`ifconfi'g'`

Reply:
The main point of this bypass is using quotes at the beginning of the payload. You can make the final quotes optional in your suggestion, and retain the quotes at the beginning.

Reply:
But the bypass can be in any position, right?

Reply:
No, it only works in the beginning, and as azurit said, the end. I didn't take into account the closing quotes. 

Azurit, I'll have to fix this on my side, otherwise there will errors regarding crs-toolchain update command. Can you suggest the final payload.

Reply:
It works anywhere.
```
❯ if'con'fig
lo0: flags=8049<UP,LOOPBACK,RUNNING,MULTICAST> mtu 16384
	options=1203<RXCSUM,TXCSUM,TXSTATUS,SW_TIMESTAMP>
...
```

Reply:
Yes, but our rules detect it if the quotes in the middle it also gets detected by my regex and rule 932240.

Reply:
`curl -H "x-format-output: txt-matched-rules" "https://sandbox.coreruleset.org/?text%3Dif%27con%27fig"` gives me no match. Is it detected?

Reply:
Yes, in my local tests two rules get triggered with your payload: 932240 and 932290.

Reply:
> Yes, but our rules detect it. If the quotes are in the middle it also gets detected by my regex and rule 932240.

So why we need a new rule if there already is one which detects it in the middle of the command?

Reply:
Okay, I checked again and only this rule gets triggered.